test/swar/BasicOperations.cpp

```diff
@@ -258,200 +335,173 @@ static_assert(0x0808'0808 == u32(broadcast<8>(SWAR<8, u32>(0x0000'0008))));
-
-static_assert(1 == lsbIndex(1<<1));
-static_assert(3 == lsbIndex(1<<3));
-static_assert(5 == lsbIndex(1<<5));
-static_assert(8 == lsbIndex(1<<8));
-static_assert(17 == lsbIndex(1<<17));
-static_assert(30 == lsbIndex(1<<30));
+static_assert(0x0808'0808'0808'0808ull ==
+              u64(broadcast<8>(SWAR<8, u64>(0x0000'0000'0000'0008ull))));
 
+static_assert(1 == lsbIndex(1 << 1));
+static_assert(3 == lsbIndex(1 << 3));
+static_assert(5 == lsbIndex(1 << 5));
+static_assert(8 == lsbIndex(1 << 8));
+static_assert(17 == lsbIndex(1 << 17));
+static_assert(30 == lsbIndex(1 << 30));
 
 /*These tests were not catching errors known to have been present
-static_assert(0x80880008 == greaterEqual<3>(SWAR<4, uint32_t>(0x3245'1027)).value());
-static_assert(0x88888888 == greaterEqual<0>(SWAR<4, uint32_t>(0x0123'4567)).value());
-static_assert(0x88888888 == greaterEqual<0>(SWAR<4, uint32_t>(0x7654'3210)).value());
-static_assert(0x00000008 == greaterEqual<7>(SWAR<4, uint32_t>(0x0123'4567)).value());
-static_assert(0x80000000 == greaterEqual<7>(SWAR<4, uint32_t>(0x7654'3210)).value());
+static_assert(0x80880008 == greaterEqual<3>(SWAR<4,
+uint32_t>(0x3245'1027)).value()); static_assert(0x88888888 ==
+greaterEqual<0>(SWAR<4, uint32_t>(0x0123'4567)).value());
+static_assert(0x88888888 == greaterEqual<0>(SWAR<4,
+uint32_t>(0x7654'3210)).value()); static_assert(0x00000008 ==
+greaterEqual<7>(SWAR<4, uint32_t>(0x0123'4567)).value());
+static_assert(0x80000000 == greaterEqual<7>(SWAR<4,
+uint32_t>(0x7654'3210)).value());
 */
 
+#define GE_MSB_TEST(left, right, result)                                       \
+  static_assert(result == greaterEqual_MSB_off<4, u32>(SWAR<4, u32>(left),     \
+                                                       SWAR<4, u32>(right))    \
+                              .value());
 
-#define GE_MSB_TEST(left, right, result) static_assert(result == greaterEqual_MSB_off<4, u32>(SWAR<4, u32>(left), SWAR<4, u32>(right)).value());
-
-GE_MSB_TEST(
-    0x1000'0010,
-    0x0111'1101,
-    0x8000'0080)
-GE_MSB_TEST(
-    0x4333'3343,
-    0x4444'4444,
-    0x8000'0080)
-GE_MSB_TEST(
-    0x0550'0110,
-    0x0110'0550,
-    0x8888'8008)
-GE_MSB_TEST(
-    0x4771'1414,
-    0x4641'1774,
-    0x8888'8008)
-GE_MSB_TEST(
-    0x0123'4567,
-    0x0000'0000,
-    0x8888'8888)
-GE_MSB_TEST(
-    0x0123'4567,
-    0x7777'7777,
-    0x0000'0008)
-GE_MSB_TEST(
-    0x0000'0000,
-    0x0123'4567,
-    0x8000'0000)
-GE_MSB_TEST(
-    0x7777'7777,
-    0x0123'4567,
-    0x8888'8888)
+GE_MSB_TEST(0x1000'0010, 0x0111'1101, 0x8000'0080)
+GE_MSB_TEST(0x4333'3343, 0x4444'4444, 0x8000'0080)
+GE_MSB_TEST(0x0550'0110, 0x0110'0550, 0x8888'8008)
+GE_MSB_TEST(0x4771'1414, 0x4641'1774, 0x8888'8008)
+GE_MSB_TEST(0x0123'4567, 0x0000'0000, 0x8888'8888)
+GE_MSB_TEST(0x0123'4567, 0x7777'7777, 0x0000'0008)
+GE_MSB_TEST(0x0000'0000, 0x0123'4567, 0x8000'0000)
+GE_MSB_TEST(0x7777'7777, 0x0123'4567, 0x8888'8888)
```

Comment:
Do you think Scott did the multi-line gratuitously?

Reply:
auto-formatter, see above.

Reply:
The question I asked was to help not missing the point: whitespace is a communication channel.  Scott put this in lines so that one could easily see the three things, the two values under comparison and the result.
The problem with using auto formatting is that it destroys whitespace as a communications channel.  That's the problem.